File: Logistics.API/Logisitcs.BLL.Interfaces/ModelInterfaces/IArticleData.cs
using System;

namespace Logisitcs.BLL.Interfaces.ModelInterfaces
{
    public interface IArticleData
    {
        Guid ArticleGuid { get; set; }
        string ArticleName { get; set; }
        string Description { get; set; }
        int? Gtin { get; set; }
        string Unit { get; set; }
        double? Position { get; set; }
        string Status { get; set; }
        int? Quantity { get; set; }
        Guid BoxGuid { get; set; }
        Guid ArticleBoxAssignment { get; set; }
        DateTime? ExpiryDate { get; set; }
    }
}

File: logistics-app/src/app/models/IArticleData.ts
import { Guid } from "guid-typescript"

export interface IArticleData
{
  articleGuid: string; 
  articleName: string; 
  description: string; 
  gtin?: number; 
  unit: string; 
  position: number; 
  status: string;
  quantity: number; 
  boxGuid: string;  
  articleBoxAssignment: string;
  expiryDate?: Date; 
}